libraries/plugins/witness/witness.cpp

```diff
@@ -109,6 +109,10 @@ void witness_plugin::plugin_initialize(const boost::program_options::variables_m
          _private_keys[key_id_to_wif_pair.first] = *private_key;
       }
    }
+   if(options.count("required-participation"))
+   {
+       _required_witness_participation = options["required-participation"].as<uint32_t>()*GRAPHENE_1_PERCENT;
```

Comment:
Please check for the allowed range 0-100, and add warnings for values <10 or >90.

Reply:
added at https://github.com/bitshares/bitshares-core/pull/1787/commits/f28d9bceb9e1d3d54c734b62e827136fbc7775f1